### DISCUSSION TITLE
[Question] Secureboot on NixOS

### DISCUSSION THREAD

**Discussion by OmegaRogue** [linux-surface/linux-surface#1443] [General]
How easy is it to get Secureboot working on NixOS? I've tried to get it working with arch linux a bunch of times, MokManager always says that something is invalid, with fedora it works perfectly fine, but i cant find anything on secureboot with the surface linux lernel on nixos in the wiki, so how is it?

**Comment by OmegaRogue** [comment]
Actually, secureboot also just stopped working on fedora with refind (access violation 0x1A)

**Comment by 8bitbuddhist** [comment]
I was able to get it working on an SP9 by following this guide: https://github.com/nix-community/lanzaboote?tab=readme-ov-file

**Comment by Flameopathic** [comment]
lanzaboote works great for me on NixOS, just as 8bitbuddhist said

SP7, NixOS
